### ISSUE TITLE
Release of pretrained model on SemanticKITTI

### ISSUE THREAD

**Issue by binyang97** [Pointcept/PointTransformerV3#102]
Thank you for the excellent work and release of the source code! I just want to ask if you have any plan to release the checkpoint trained on SemanticKITTI? I want to reproduce some baseline results for my recent project and it would be very helpful if you can share the file.

**Comment by Gofinge** [comment]
Yes, yes, yes. Might need to wait until after this CVPR deadline. I will have a big update at the end of this year. I cannot access some old experiment records due to the change of internship company.

**Comment by binyang97** [comment]
Thanks for the reply. Looking forward to the update and wishing you good luck for the coming CVPR :)

**Comment by lucky9-cyou** [comment]
Hello, I would like to ask if there are any updates regarding the model?

**Comment by Gofinge** [comment]
I am waiting for the law review process done by my current intern company and then release the next version of Pointcept with new work and configs with more datasets. We used to expect to finish the process at the beginning of Jan, yet the process is much lower than we expected.
